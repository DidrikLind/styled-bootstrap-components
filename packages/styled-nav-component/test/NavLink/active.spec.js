import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { NavLink } from '../../src';

test('Styles matches NavLink active', () => {
  const tree = renderer.create(<NavLink active />).toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule('color', '#495057');
  expect(tree).toHaveStyleRule('background-color', '#fff');
});
