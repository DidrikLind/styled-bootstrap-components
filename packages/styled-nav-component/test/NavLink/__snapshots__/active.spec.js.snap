// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles matches NavLink active 1`] = `
.c0 {
  display: block;
  padding: 0.5rem;
  border: 1px solid transparent;
  border-top-left-radius: 0.25rem;
  border-top-right-radius: 0.25rem;
  color: #007bff;
  color: #495057;
  background-color: #fff;
}

.c0:hover,
.c0:focus {
  text-decoration: none;
  color: #0056b3;
}

<a
  className="c0 sc-bdVaJa eRiqeT"
/>
`;
